### PR TITLE
2.0.9 release

### PR DESCRIPTION
**Pull request by rjbou**

### CODE REVIEW

File: opam-2-0-9.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+authors: [
+  "Raja Boujbel - OCamlPro" {"mailto:raja.boujbel(à)ocamlpro.com"}
+  "Louis Gesbert - OCamlPro" {"mailto:louis.gesbert(à)ocamlpro.com"}
+]
+date: "2021-07-30"
+--BODY--
+
+We are pleased to announce the minor release of [opam 2.0.9](https://github.com/ocaml/opam/releases/tag/2.0.9).
+
+This new version contains some [backported](https://github.com/ocaml/opam/pull/4547) fixes.
```
Comment (dra27) [inline opam-2-0-9.md:11]:
```suggestion
This new version contains some [back-ported](https://github.com/ocaml/opam/pull/4547) fixes.
```

File: opam-2-0-9.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## New features
+  * Back-ported ability to load upgraded roots read-only; allows applications compiled with opam-state 2.0.9 to load a root which has been upgraded to opam 2.1 [[#4636](https://github.com/ocaml/opam/issues/4636)]
+  * macOS sandbox now supports `OPAM_USER_PATH_RO` for adding a custom read-only directory to the sandbox [[#4589](https://github.com/ocaml/opam/issues/4589), [#4609](https://github.com/ocaml/opam/issues/4609)]
+  * `OPAMROOT` and `OPAMSWITCH` now reflect the `--root` and `--switch` parameters in the package build [[#4668](https://github.com/ocaml/opam/issues/4668)]
+  * When built with opam-file-format 2.1.3+, opam-format 2.0.x displays better errors for newer opam files [[#4394](https://github.com/ocaml/opam/issues/4394)]
+
+## Bug fixes
+  * Linux sandbox now mounts _host_ `$TMPDIR` read-only, then sets the _sandbox_ `$TMPDIR` to a new separate tmpfs. **Hardcoded `/tmp` access no longer work if `TMPDIR` points to another directory** [[#4589](https://github.com/ocaml/opam/issues/4589)]
```
Comment (dra27) [inline opam-2-0-9.md:20]:
```suggestion
  * Linux sandbox now mounts _host_ `$TMPDIR` read-only, then sets the _sandbox_ `$TMPDIR` to a new separate tmpfs. **Hardcoded `/tmp` access no longer works if `TMPDIR` points to another directory** [[#4589](https://github.com/ocaml/opam/issues/4589)]
```

File: opam-2-0-9.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  * Linux sandbox script no longer makes `PWD` read-write on remove actions [[#4589](https://github.com/ocaml/opam/issues/4589)]
+  * Lint W59 and E60 no longer trigger for packages flagged `conf` [[#4549](https://github.com/ocaml/opam/issues/4549)]
+  * Reduce the length of temporary file names for pin caching to ease pressure on Windows [[#4590](https://github.com/ocaml/opam/issues/4590)]
+  * Security: correct quoting of arguments when removing switches [[#4707](https://github.com/ocaml/opam/issues/4707)]
+  * Stop advertising the removed option `--compiler` when creating local switches [[#4718](https://github.com/ocaml/opam/issues/4718)]
+  * Pinning no longer fails if the archive's opam file is malformed [[#4580](https://github.com/ocaml/opam/issues/4580)]
+  * Fish: stop using deprecated `^` syntax to fix support for Fish 3.3.0+ [[#4736](https://github.com/ocaml/opam/issues/4736)]
+
+
+A more complete [release note](https://github.com/ocaml/opam/releases/tag/2.0.9) is available.
```
Comment (dra27) [inline opam-2-0-9.md:32]: Technically speaking it's the same release note 🙂

File: opam-2-0-9.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    opam update; opam install opam-devel
+    ```
+
+   (then copy the opam binary to your PATH as explained, and don't forget to run `opam init --reinit -ni` to enable sandboxing if you had version 2.0.0~rc manually installed or to update your sandbox script)
+
+3. From source, manually: see the instructions in the [README](https://github.com/ocaml/opam/tree/2.0.9#compiling-this-repo).
+
+We hope you enjoy this new minor version, and remain open to [bug reports](https://github.com/ocaml/opam/issues) and [suggestions](https://github.com/ocaml/opam/issues).
+
+> NOTE: this article is cross-posted on [opam.ocaml.org](https://opam.ocaml.org/blog/) and [ocamlpro.com](http://www.ocamlpro.com/category/blog/), and published in [discuss.ocaml.org](https://discuss.ocaml.org/t/ann-opam-2-0-8-release/7242). Please head to the latter for the comments!
```
Comment (dra27) [inline opam-2-0-9.md:58]: Needs updating/removing

**Review by dra27 — COMMENTED** [review]
Couple of typos and notes only

File: opam-2-0-9.md (hunk 4 above)
Comment (rjbou) [reply opam-2-0-9.md:58]: removing

File: opam-2-0-9.md (hunk 3 above)
Comment (rjbou) [reply opam-2-0-9.md:32]: yes, there is just the greeting that are in the release not only ^^

File: opam-2-0-9.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+
+---
+
+Installation instructions (unchanged):
+
+1. From binaries: run
+
+    ```
+    sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh --version 2.0.9)
```
Comment (kit-ty-kate) [inline opam-2-0-9.md:43]:
```suggestion
    sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh) --version 2.0.9
```

File: opam-2-0-9.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+
+---
+
+Installation instructions (unchanged):
+
+1. From binaries: run
+
+    ```
+    sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh) --version 2.0.9
```
Comment (kit-ty-kate) [inline opam-2-0-9.md:43]:
```suggestion
    bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh) --version 2.0.9"
```
`<()` is non-POSIX bash syntax and will not work for users using `sh` or other non-bash derived shells.
I've also added `-f` and `-S` to curl to be more safe for users. e.g.
```
opam@8dbac0f63f40:~$ bash -c "sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/blah.sh) --version 2.0.9"
/dev/fd/63: 1: /dev/fd/63: 404:: not found
opam@8dbac0f63f40:~$ bash -c "sh <(curl -fsL https://raw.githubusercontent.com/ocaml/opam/master/shell/blah.sh) --version 2.0.9"
opam@8dbac0f63f40:~$ bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/blah.sh) --version 2.0.9"
curl: (22) The requested URL returned error: 404
```

Comment (kit-ty-kate) [reply opam-2-0-9.md:43]: (this was partly taken from homebrew: https://brew.sh/)

File: opam-2-0-9.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+title: "opam 2.0.9 release"
+authors: [
+  "Raja Boujbel - OCamlPro" {"mailto:raja.boujbel(à)ocamlpro.com"}
+  "Louis Gesbert - OCamlPro" {"mailto:louis.gesbert(à)ocamlpro.com"}
+]
+date: "2021-07-30"
```
Comment (rjbou) [inline opam-2-0-9.md:6]:
```suggestion
date: "2021-08-03"
```

**Pull request merged**